spec/features/admin/admin_deactivates_user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+feature "Admin deactivates a user" do
+  scenario "admin visits dashboard" do
+
+    admin_role = create(:role, title: "admin")
+    admin = create(:user)
+    admin.roles << admin_role
+
+    requester_role = create(:role, title: "requester")
+    requester = create(:user)
+    requester.roles << requester_role
+
+    visit login_path
+    fill_in :email, with: admin.email
```

Comment:
We should refactor at some point to use the method that returns admin to all descendants of application controller, rather than testing login again here.